qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -170,10 +254,47 @@ def run(
             api_job_share_level = ApiJobShareLevel.NONE
 
         validate_job_tags(job_tags, IBMQBackendValueError)
+
+        if isinstance(circuits, (QasmQobj, PulseQobj)):
+            warnings.warn("Passing a Qobj to Backend.run is deprecated and will "
```

Comment:
It seems that `ManagedJobSet` was not adapted to the v2 provider interface: it still assembles `qobj` for `ManagedJob`. Thus, with the latest `qiskit-ibmq-provider`, the use of `IBMQJobManager` now yields two related warnings for each and every job submission which with many jobs becomes somewhat overwhelming. Is there hope to get this cleaned up in the next release, please?